components/style/themes/default.less

```diff
@@ -386,7 +394,8 @@
 @card-padding-base: 24px;
 @card-padding-wider: 32px;
 @card-actions-background: @background-color-light;
-@card-shadow: 0 2px 8px rgba(0, 0, 0, .09);
+@card-shadow: 0 2px 8px fade(@black, 9%);
+@card-background: #cfd8dc;
```

Comment:
I can't find `@card-background`'s usage.

Reply:
It is used in [components/card/style/index.less#L8](https://github.com/Nouzbe/ant-design/blob/feature-3.8.0/components/card/style/index.less#L8).